docs/examples/config-private-ghcr.yaml

```diff
@@ -0,0 +1,49 @@
+# To use a private policy image you need to fill the following configuation with the required information:
+# 1. Github PAT with the available rights to access private packages.
+# 2. Your organization
+# 3. Your policy image name and label
+---
+logging:
+  prod: true
+  log_level: debug
+
+directory_service:
+  edge:
+    db_path: ./pkg/testing/assets/eds-citadel.db
```

Comment:
should this be the same as the previous example?  `${TOPAZ_DIR}/db/directory.db`